pkg/solana/txm/txm.go

```diff
@@ -765,6 +773,14 @@ func (txm *Txm) Enqueue(ctx context.Context, accountID string, tx *solanaGo.Tran
 		lastValidBlockHeight: txLastValidBlockHeight,
 	}
 
+	err = txm.txs.New(msg)
+	// No-op if transaction already exists for provided ID
+	if err != nil && errors.Is(err, ErrIDAlreadyExists) {
```

Comment:
This is just sanity checking right? We already checked if Tx exists above in this function. So this condition likely won't be true.